.eslintrc

```diff
@@ -14,14 +14,15 @@
       "number": true,
       "string": true
     }],
-    "no-unused-expressions": [2, {
+    "no-unused-expressions": [0, {
```

Comment:
Yah we should probs set a temp bypass if needed. Seems too broad to change generally. I think the _ in arrow functions is normally unused, but that should be a special case we accept I think.